traja/models/nn.py

```diff
@@ -69,6 +68,29 @@ def get_timeseries_data_loaders(data_frame, sequence_length, train_fraction, bat
 
     return train_loader, validation_loader
 
+def get_transformed_timeseries_dataloaders(data_frame, sequence_length, train_fraction, batch_size, shift):
+    
```

Comment:
Can you please document this method, what does it do? And typo annotations for arguments and return variables would be helpful.

Reply:
I committed the requested changes now. The PR is targetted towards the `upstream` branch `NeuralNetworks` and not towards `master` branch ;)